### PR TITLE
Add basic support for linear-gradient()

### PR DESCRIPTION
**Pull request by mthadley**
Hello everyone,

I was trying out `elm-css` on one of my projects and needed to use a `linear-gradient`. I wasn't able to find it anywhere so I thought I'd try adding it!

I attempted to follow the pre-existing patterns, however I think I may have missed or misinterpreted a few things. In particular, I'm not sure if `ColorStop`/`stop`/`stopAt` is exactly what we want.

Thanks!

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,5 +47,5 @@ install:
   - cd ..
 
 script:
-  - elm-format --validate .
+  - elm-format --validate src tests examples readme-example
```
Comment (eeue56) [inline .travis.yml]: I'd make this change as part of a seperate PR.

Comment (mthadley) [inline .travis.yml]: Yea, that makes sense. I just wanted to move this PR into the green.

Comment (mthadley) [inline .travis.yml]: Sent as #232

**Comment by rtfeldman** [comment]
Thanks @mthadley! I'm gonna rename `stopAt` to `stop2` but otherwise, looks great! 🤘

**Pull request merged**